app/services/support_interface/send_initial_emails.rb

```diff
@@ -0,0 +1,30 @@
+module SupportInterface
+  class SendInitialEmails
+    attr_accessor :ucas_match
+
+    def initialize(ucas_match:)
+      self.ucas_match = ucas_match
+    end
+
+    def call
+      return false if ucas_match.initial_emails_sent?
+
+      return ::SendInitialEmailsMultipleAcceptances.new(ucas_match).call if ucas_match.application_accepted_on_ucas_and_accepted_on_apply?
+
+      application_choice = application_choice_for(ucas_match)
+
+      CandidateMailer.ucas_match_initial_email_duplicate_applications(application_choice).deliver_later
```

Comment:
Should this and the following few lines be turned into a service with a descriptive name and a spec?

Reply:
Done.